.travis.yml

```diff
@@ -15,6 +15,8 @@ matrix:
       os: linux
 
 before_install:
+  - echo -e "{\n  \"ipv6\":true,\n  \"fixed-cidr-v6\":\"2001:db8:1::/64\"\n}" | sudo tee /etc/docker/daemon.json
```

Comment:
Maybe use the multiline string syntax of yaml to make it less cluttered with `\n`, and more clear?

Reply:
Or just remove the linebreaks. They're not needed for JSON and there is no gain from having a pretty formatted file here.
The shell argument can be single quoted as well, this way you don't need any escapes at all.
```bash
echo '{ "ipv6": true, "fixed-cidr-v6": "2001:db8:1::/64" }' | sudo tee /etc/docker/daemon.json`
```

Reply:
true, didn't realized it was json ^^